Makefile

```
@@ -72,6 +72,16 @@ coverage: ## Run tests with coverage report
 	go tool cover -html=coverage.out -o coverage.html
 	@echo "Open coverage.html in your browser for details"
 
+local-cluster-smoke: ## Smoke-test local-cluster manager against real kind (manual; pre-release)
```

checkmake (0.3.2)

[warning] Target body for "local-cluster-smoke" exceeds allowed length of 5 lines (9).

(maxbodylength)